### ISSUE TITLE
Not running

### ISSUE THREAD

**Issue by justin-pierce** [fabiospampinato/vscode-terminals#49]
Recently my Terminals (triggered by Commands) stopped working. The Terminal window is created, but it isn't given focus and the command isn't actually executed. Seems like a loading issue (maybe broken by recent vs code update) -- if I change the name of the command in the settings file, it will run.

**Comment by fabiospampinato** [comment]
Are you on VS Code Insiders?

**Comment by justin-pierce** [comment]
Nope. And just checked, no recent updates (running on 1.43.0 macOS). So no idea what broke -- had worked fine for years then just stopped a few days ago. Will let you know if I find way to reproduce it.

**Comment by wallzero** [comment]
I have the same issue on Linux since VSCode version 1.56. Terminals are created but not focused/active. I created keyboard shortcuts to navigate between terminals which still work but I then have to manually mouse click on a terminal at least once in order to type anything.

**Comment by fabiospampinato** [comment]
Can anybody check if v2.0.2 still has this problem?
